libs/watt/src/lib/utils/date/watt-date.pipe.ts

```diff
@@ -42,10 +42,16 @@ export class WattDatePipe implements PipeTransform {
   ): string | null {
     if (!input) return null;
 
-    return input instanceof Date || typeof input === 'string'
-      ? dayjs(input).tz(timeZone).format(formatStrings[format])
-      : typeof input === 'number'
-        ? dayjs(new Date(input)).tz(timeZone).format(formatStrings[format])
-        : `${this.transform(input.start, format)} ― ${this.transform(input.end, format)}`;
+    if (input instanceof Date || typeof input === 'string') {
+      return dayjs(input).year() === 10000 // Max Date is the same as null so dont show it
```

Comment:
nitpick: slighly changed wording hopefully making it more clear 


```suggestion
      // Treat year 1000 as `null` in the UI
      // Needed because in some cases `input.end` is set to a value that is far into the future
      // in order to signify that the value is not defined  
      return dayjs(input).year() === 10000 
```